pkg/component/watch_test.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows,!osx
```

Comment:
Skipping watch test build on windows and macOS. It will be revised in https://github.com/openshift/odo/issues/2570

Reply:
> Skipping watch test build on windows and macOS. It will be revised in #2570

It would be good to add this as a comment to all skipped files.